exploration/start_remote_environment.py

```diff
@@ -64,10 +61,17 @@
     server_credentials_paths = (args.server_certificate, args.server_private_key, args.root_certificate)
     enable_rendering = args.enable_rendering
 
-    example_environment = gym.make("Acrobot-v1", render_mode="rgb_array")
-
     server = start_as_remote_environment(
-        local_environment=example_environment,
+        default_args={
```

Comment:
I'm not 100% happy with merging args and settings here but splitting them up just adds complexity with minimal benefit either.

Reply:
I would say readability is a nice benefit. Splitting them would be also easy to implement, since `remote_environment_manager.create_gym_environment` has a logical split.